articles/qdk-relnotes.md

```diff
@@ -19,6 +19,14 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update-qdk).
 
+## Version 0.15.2103.133969
+
+
+*Release date: March 30th, 2021*
+
+- Released QIR emission as experimental feature.
+- Added support for multiple entry points.
```

Comment:
@bettinaheim Should we specify that this is _command line_ support for multiple entry points, and that this isn't supported for IDE builds?